base/expr.jl

```diff
@@ -88,9 +88,9 @@ julia> macro m()
 julia> M.f()
 (1,2)
 ```
-With @macroexpand the expression expands where @macroexpand appears in the code (module M).
-With macroexpand the expressions expands in the current module where the code was finally called.
-Note that when calling macroexpand or @macroexpand directly from the REPL, both of these contexts coincide, hence there is no difference.
+With `@macroexpand` the expression expands where `@macroexpand` appears in the code (module `M`).
+With `macroexpand` the expressions expands in the current module where the code was finally called (REPL).
```

Comment:
The `julia>` in the example implies that it runs in the REPL, doesn't it? And the "(module `M`)" in the sentence before also makes sense only  in context of the example. So I'd say either leave as is or rephrase as "(module `M` in the example)" and "(REPL in the example)" (which I could well do in #18784).